internal/server/iigointernal/judiciary.go

```diff
@@ -133,7 +133,7 @@ func (j *judiciary) appointNextPresident(monitoring shared.MonitorResult, curren
 			return j.gameState.PresidentID, errors.Errorf("Insufficient Budget in common Pool: appointNextPresident")
 		}
 		election.ProposeElection(shared.President, electionsettings.VotingMethod)
-		election.OpenBallot(electionsettings.IslandsToVote, iigoClients)
+		election.OpenBallot(electionsettings.IslandsToVote, iigoClients, allIslands)
```

Comment:
just an extra check that only non-dead islands are passed in, right?

Reply:
allIslands is passed in from orchestration where it is populated with only alive client IDs